consul/datadog_checks/consul/consul.py

```diff
@@ -72,6 +72,12 @@ class ConsulCheck(AgentCheck):
 
     STATUS_SEVERITY = {AgentCheck.UNKNOWN: 0, AgentCheck.OK: 1, AgentCheck.WARNING: 2, AgentCheck.CRITICAL: 3}
 
+    HTTP_CONFIG_REMAPPER = {
+        'client_cert_file': {'name': 'tls_cert', 'default': None, 'invert': False},
+        'private_key_file': {'name': 'tls_private_key', 'default': None, 'invert': False},
```

Comment:
```suggestion
        'private_key_file': {'name': 'tls_private_key'},
```

Seems default and invert are not needed.